ingestion/ingest-wordpress.py

```diff
@@ -0,0 +1,82 @@
+import base64
+import json
+import os.path
+import requests
+
+from flask import Flask, has_request_context, request
+from flask_restful import Api, Resource, reqparse
+from git import Repo
+
+
+ROOT_PATH = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..'))
+GIT_REPO_PATH = f'{ROOT_PATH}/.git'
+COMMIT_MESSAGE = 'Update shared repository'
+
+ENABLE_PUSH = False # Set to false when just testing
+
+# TODO: Unit testing, end-to-end testing
+
+# Push shared repository to Git if any files changed
+# TODO: Make this a shared library with ingest-google-docs
+def git_push():
+    repo = Repo(GIT_REPO_PATH)
+    t = repo.head.commit.tree
+    repo.index.add(["shared_directory"])
+    if repo.git.diff(t):
+        repo.index.commit(COMMIT_MESSAGE)
```

Comment:
Should we add a dynamically generated "version/timestamp" in the commit message that we can then parse to more-easily figure out the version of the file?

We could also just use the commit hash to figure out whether there are any newer versions of the file or not.

I was thinking we can either add this version/timestamp to the commit message of the translated documents as well so the wordpress updater can compare and see if the translations are for the latest version of the file or not.

Reply:
Right now I'm storing a timestamp field 'modified_gmt' in the json.

I'm open to putting any type of timestamp, hash, etc in whatever location would be most convenient for Serge and our TMS.